redis/redis.go

```diff
@@ -130,9 +130,10 @@ func (s *Storage) Create(name string, capacity uint, rate time.Duration) (leakyb
 
 // New initializes the connection to redis.
 func New(network, address string) (*Storage, error) {
+	timeout := time.Duration(5000 * millisecond) // 5 seconds
 	s := &Storage{
 		pool: redis.NewPool(func() (redis.Conn, error) {
-			return redis.Dial(network, address)
+			return redis.Dial(network, address, redis.DialReadTimeout(timeout))
```

Comment:
Maybe add comment here about we need extra config if we change the usage pattern in the library?